client-src/elements/chromedash-guide-editall-page.ts

```diff
@@ -280,7 +285,7 @@ export class ChromedashGuideEditallPage extends LitElement {
         <chromedash-form-field
           name=${field}
           index=${index}
-          stageId=${stageId}
+          stageId=${ifDefined(ifDefined(stageId))}
```

Comment:
Do we need the nested `ifDefined()` invocation here?

Reply:
seems like we dont need it as i tried it out. It's nested because my editor showed an error of this potentially being "null" string. I'll remove them